README.md

```diff
@@ -90,85 +90,197 @@ Here the list of existing promise adapters:
 This bundle can be use with [GraphQLBundle](https://github.com/overblog/GraphQLBundle).
 Here an example:
 
-* Bundle config
+* First create your service. We will use the Webonyx promise adapter
 
 ```yaml
-#graphql
+#config/services.yaml
+services:
+    graphql_promise_adapter:
+        class: Overblog\DataLoader\Promise\Adapter\Webonyx\GraphQL\SyncPromiseAdapter
+        public: true
+        
+    
+    ###
+    # Force the overblog promise adapter to use the webonyx adapter
+    ###
+    Overblog\PromiseAdapter\PromiseAdapterInterface:
+        class: Overblog\PromiseAdapter\Adapter\WebonyxGraphQLSyncPromiseAdapter
+        arguments:
+            - "@graphql_promise_adapter"
+    
```

Comment:
Seems very verbose, this was the only configuration that I needed:

```yaml
    Overblog\DataLoader\Promise\Adapter\Webonyx\GraphQL\SyncPromiseAdapter: ~
    Overblog\PromiseAdapter\Adapter\WebonyxGraphQLSyncPromiseAdapter: ~
    Overblog\PromiseAdapter\PromiseAdapterInterface: '@Overblog\PromiseAdapter\Adapter\WebonyxGraphQLSyncPromiseAdapter'
```

Why not just register this out of the box?